db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_06_04_172236) do
+ActiveRecord::Schema.define(version: 2019_06_18_200946) do
```

Comment:
It strikes me as odd that this value changed yet this PR doesn't seem to add/run any migrations.  Its possible I'm missing a piece WRT to GraphQL, but I think this can be removed.